docs/conf.py

```diff
@@ -94,6 +94,12 @@
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path.
 exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', 'tutorials/*', 'graveyard/*']
+# add temporary unused files
```

Comment:
I suggest to remove if-else statement above (`PRODUCT_VERSION == 'Intel'`), it looks outdated